src/SignInPage/index.js

```diff
@@ -260,4 +253,5 @@ class SignIn extends React.Component<Props, State> {
   }
 }
 
-export default SignIn;
+export const RawSignIn = SignIn;
```

Comment:
hmm `RawSignIn` is weird naming I think. Maybe `NativeSignIn` or `EmailSignIn` instead?

Reply:
Maybe those should also be changed :stuck_out_tongue_winking_eye:
Like I don't feel the word "raw" describes "email" very well.